### PR TITLE
travis.yml: travis buddy

### PR DESCRIPTION
**Pull request by commitay**
https://github.com/caskroom/homebrew-cask/pull/45294#issuecomment-377112070

https://github.com/caskroom/homebrew-cask/pull/45287#issuecomment-377111444

### CODE REVIEW

**Review by vitorgalvao — APPROVED** [review]

**Comment by commitay** [comment]
Template still needs to be adjusted.

https://github.com/caskroom/homebrew-cask/pull/45287#issuecomment-377111444

Not sure what changes we need to make.

**Comment by bluzi** [comment]
Hey @commitay, just wanted you to know what you saw at https://github.com/caskroom/homebrew-cask/pull/45287#issuecomment-377111444 was a problem on our side, and that it's fixed now. 

Let me know if you occur further issues with TravisBuddy.

**Comment by commitay** [comment]
Thanks @bluzi!

File: .travis.yml [hunk 1]
```diff
@@ -37,6 +37,9 @@ notifications:
   webhooks:
     urls:
       - "https://webhooks.gitter.im/e/712d699360b239db14a5"
-    on_success: change
+      - "https://www.travisbuddy.com/"
+    on_success: never
     on_failure: always
     on_start: never
+    on_cancel: never
+    on_error: never
```
Comment (reitermarkus) [inline .travis.yml]: What's the difference between `on_failure` and `on_error`?

Comment (commitay) [reply .travis.yml]: Error is an issue with the CI itself, failure is on our side.

Comment (bluzi) [reply .travis.yml]: Failure means the tests failed, as one of the scripts defined in `.travis.yml` exited with non-zero code. (In your case it's `. ci/travis/script.sh`)

An error means the actual builds failed to execute. (For instance, in a case of a missing dependency or any other error in the build process)

Comment (reitermarkus) [reply .travis.yml]: Thanks, I thought it was something like that.

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**